src/modules/auth/router.js

```diff
@@ -0,0 +1,26 @@
+const AuthService = require('./service')
+const cookie = require('fastify-cookie')
+const jwt = require('jsonwebtoken')
+const { JWT_SECRET } = process.env
```

Comment:
Relative imports luôn để dưới absolute imports

```suggestion
const cookie = require('fastify-cookie')
const jwt = require('jsonwebtoken')
const AuthService = require('./service')

const { JWT_SECRET } = process.env
```